lab-kyler/test/note-route-test.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+const request = require('superagent');
+const expect = require('chai').expect;
+
+//var testJoke = {setup: 'I just flew in from Chernobyl', punchline: 'and boy, are my arms legs!'};
+//var testJoke = {setup: 'Why did the console player cross the road?', punchline: 'To render the buildings on the other side.'};
+//var testJoke = {setup: 'My physics teacher said I had potential', punchline: 'then he pushed me down the stairs.'};
+var testJoke = {setup: 'Sting was kidnapped last night!', punchline: 'The Police are looking for a lead.'};
+
+describe('Route: /api/joke', function() {
+
+  describe('GET-ing a nonexistent joke ID', function() {
+    it('should return status 404 and body "not found"', function (done) {
+      request.get('localhost:2000/api/joke?id=abc123')
```

Comment:
Make sure that the PORT you have declared in your server is the same PORT you are using in the URL you are passing your http methods for your tests.

Your tests passed for me once I changed the PORT to 2001 and included the server.js